solr/contrib/ltr/example/libsvm_formatter.py

```diff
@@ -71,22 +72,29 @@ def convertLibSvmModelToLtrModel(self,libSvmModelLocation, outputFile, modelName
                 convertedOutFile.write('\n\t\t}\n\t}\n}')
 
 def _writeRankSVMPairs(listOfFv,output):
```

Comment:
Done, also renamed some of the other params that functions take in to features since those are the interfaces that people would look at.  Didn't rename some of the internal variables though since that might make the lines long, and they were interacting with variables named featureVector, so I thought it was explanatory enough.